### PR TITLE
Add revision kind to RevisionInfo

### PR DESCRIPTION
**Pull request by timwangmusic**
The `kind` field is missing in `RevisionInfo`. The field is important because users can use this info for a general description of what are changed in the revision.

### CODE REVIEW

File: changes.go [hunk 1]
```diff
@@ -7,6 +7,19 @@ import (
...
+//
+// Gerrit API docs: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#revision-info
+type RevisionKind string
+
+const (
+	Rework                 RevisionKind = "REWORK"
+	TrivialRebase          RevisionKind = "TRIVIAL_REBASE"
+	MergeFirstParentUpdate              = "MERGE_FIRST_PARENT_UPDATE"
+	NoCodeChange                        = "NO_CODE_CHANGE"
+	NoChange                            = "NO_CHANGE"
```
Comment (chaodaiG) [inline changes.go:20]: unrelated change?

Comment (chaodaiG) [reply changes.go:20]: or should they also be `RevisionKind`?

Comment (timwangmusic) [reply changes.go:20]: After running `gofmt -s` these repetitions are gone, I think these are simplifications?

Comment (timwangmusic) [reply changes.go]: Done.

**Review by chaodaiG — APPROVED** [review]

**Review by andygrunwald — APPROVED** [review]
This looks good.

**Comment by andygrunwald** [comment]
Thanks a lot, @timwangmusic. Also, thanks for your review @chaodaiG, much appreciated.
